### DISCUSSION TITLE
Move and use the positive signal only

### DISCUSSION THREAD

**Discussion by rroblik** [fredlcore/BSB-LAN#697] [General]
hello everyone! 

i have wifi range problems and also freezing of my arduino. i would like to make my installation more resilient by first switching from wifi to cable. for that i think i will move my bsb lan card from the heat pump to my electrical panel. to avoid having to re-run a cable (which is complicated) i noticed that i had a ground wire available on the cable of the outdoor temperature sensor (which does not use the ground). also is it possible to simply use this wire to transmit the positive signal from my x86 socket (in green on the photo) and to use another neutral from my installation for the negative terminal (in red)? and on the side of the BSB card also use a negative signal of some kind? 

sorry for my bad english! thanks
![IMG_9225](https://github.com/user-attachments/assets/e16da7b5-705e-4f2c-9de1-899db45cbc09)

**Comment by fredlcore** [comment]
I'm always a bit amused about these kind of questions, because what would people expect from me when they ask these questions? That I declare: "Sure, it's 100% safe and will never ever do anything harmful to your heating system!"?
Of course you can try it if you think it could make sense, but even if I was an electrical engineer (which I'm not), I wouldn't give you an authoritative answer for a heating system that I don't own, let alone have comlete information about.
Bottom line: If you really feel it's worth any potential risk then go ahead, but I would seriously doubt that the added convenience is worth any kind of risk.
